### DISCUSSION TITLE
Intégration de nouveaux membres au sein du PSC ?

### DISCUSSION THREAD

**Discussion by jerbou** [mviewer/mviewer#564] [Q&A] [answered]
Bonjour,

Nous utilisons Mviewer et Mviewer Studio, avec certaines spécificités lié à notre plateforme.
Nous souhaitons contribuer à ces deux logiciels dans la mesure de nos moyens, et de ce manière intégré à la communauté.

Merci,
A vous lire,

Jérôme Boutet 
IDéO BFC

**Comment by bchartier** [comment]
Je colle ici un élément échangé initialement via un message privé :

> Le PSC n’a pas formalisé l’intégration de nouveaux membres, le sujet et ta candidature seront évoqués lors de la prochaine réunion (22/02).

**Comment by lecault** [comment]
Bonjour Jérôme et merci de ta candidature.

@PSC, est-ce qu'on prévoit un temps d'échange avec Jérôme lors du prochain PSC avant analyse de sa candidature ?

Loïc

**Reply by bchartier** [reply]
Oui.

**Comment by jerbou** [comment]
Merci à vous.
Je me rend disponible pour un échange, du 22/02
Pourriez vous me communiquer un horaire et un mode d'échange (visio ou tel) 
vous avez mon courriel) :)

Merci,

**Comment by lecault** [comment]
Je te transmets ça

**Reply by jerbou** [reply]
merci ! C'est noté.

**Comment by lecault** [comment, marked as answer]
Discussion close, bienvenue Jérôme

**Reply by jerbou** [reply]
Merci pour votre accueil, dans la bonne humeur studieuse :)

A bientôt,
